qemu/tests/cfg/qsd_live_snapshot.cfg

```diff
@@ -0,0 +1,37 @@
+- qsd_live_snapshot:
+    type = qsd_live_snapshot
+    start_vm = no
+    images + = "stg1"
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    base_tag = "stg1"
+    snapshot_tag = sn1
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+    device = "fmt_stg1"
+    format = qcow2
+    rebase_mode = unsafe
+    only Linux
+    !Host_RHEL.m7:
+        node = "fmt_stg1"
+        overlay = "drive_sn1"
+        qemu_force_use_drive_expression = no
+
+    # QSD attributes
+    qsd_namespaces = "qsd1"
+    qsd_images_qsd1 = "stg1"
```

Comment:
qsd_namespaces = qsd1 qsd2
qsd_images_qsd2 = sn1

Then we can create two different qsd daemons, one for stg1 export and the other for sn1 export, in py file, we can create two qsd objects

Reply:
Just start qsd with qsd1.